apps/web/src/stores/providerApiKeys.ts

```diff
@@ -24,7 +26,7 @@ export default function useProviderApiKeys(opts?: SWRConfiguration) {
     return data || []
   }
   const {
-    data = [],
+    data = EMPTY_ARRAY,
```

Comment:
This guy was being was being set as a different "empty array" instance on every single render, and thus triggering anything that subscribed to it as a dependency. Thanks to this, opening a new document would compute the document's metadata 50+ times while this store was still loading its data.